azakharevich/octonion/AQA_lessons/Class/person.py

```diff
@@ -0,0 +1,53 @@
+class Person:
+    def __init__(self) -> None:
+        super().__init__()
+        self.itemPrice = 0
+        self.equipArray = []
+
+
+    def addEquip(self, object):
+        self.itemPrice += object.price
+        self.equipArray.append(object)
+
+
+    def fullEquipPrice(self):
+        return self.itemPrice
+
+
+    def sortEquip(self):
+        print('\n\rSorting has been started...')
+
+        self.sortedArray = []
+
+        print(' Before:')
+        for i in range(len(self.equipArray)):
+            print('-', self.equipArray[i].weight)
+
+        while len(self.equipArray) > 0:
+            self.minEquip = self.equipArray[0]
+            self.minIndex = 0
+            for i in range(len(self.equipArray)):
+                self.currEquip = self.equipArray[i]
+                if self.currEquip.weight < self.minEquip.weight:
+                    self.minEquip = self.currEquip
+                    self.minIndex = i
+            self.sortedArray.append(self.minEquip)
+            self.equipArray.pop(self.minIndex)
+        self.equipArray = self.sortedArray
+
+        print(' After:')
```

Comment:
i would move both prints (before and after) to separate method that is called from main